src/index.js

```diff
@@ -5,12 +5,39 @@ import UrlPattern from 'url-pattern';
 
 const DEFAULT_REQUEST_METHOD = 'GET';
 
+const fetchFactoryTemplates = {
+  findOne: {
+    method: 'GET',
+  },
+  findAll: {
+    method: 'GET',
+  },
+  create: {
+    method: 'POST',
+  },
+  update: {
+    method: 'PUT',
+  },
+};
+
 const fetchFactory = {
-  create(options, methods) {
+  create(options, methods = {}) {
     this.factory = {};
 
     this.defaultOptions = options;
 
+    const templateMethods = (options.methods || []).map((methodName) => {
```

Comment:
actually, set the default to `[]` then you don't need to do this logical or expression at all